index.html

```diff
@@ -138,8 +138,8 @@
       >
       <a
         id="application-button"
-        href="/applications"
-        class="action-button element-display-remove"
+        href="/applications/index.html"
```

Comment:
- why is this required does navigating to application not render the index.html by default?